test/functional/dynamicDispatchTask.test.ts

```diff
@@ -27,1055 +27,1090 @@ const initialize = async () => {
-
-    await expect(
-      automationTimeApi.buildScheduleDynamicDispatchTask(
-        keyringPair,
-        schedule,
-        call,
-      ),
-    ).rejects.toThrow();
-  },
-  DEFAULT_TIMEOUT_PER_TEST,
-);
-
-/**
- * dynamicDispatchTask.recurring with frequency that is object fails
- */
-test(
-  "dynamicDispatchTask.recurring with nextExecutionTime that is object fails",
-  async () => {
-    const extrinsicParams = await getDynamicDispatchExtrinsicParams(
-      polkadotApi,
-      "recurring",
-    );
-    const { schedule, call } = extrinsicParams;
-    schedule.recurring.nextExecutionTime = {};
-
-    await expect(
-      automationTimeApi.buildScheduleDynamicDispatchTask(
-        keyringPair,
-        schedule,
-        call,
-      ),
-    ).rejects.toThrow();
-  },
-  DEFAULT_TIMEOUT_PER_TEST,
-);
-
-/**
- * dynamicDispatchTask.recurring with nextExecutionTime that is null fails
- */
-test(
-  "dynamicDispatchTask.recurring with nextExecutionTime that is null fails",
-  async () => {
-    const extrinsicParams = await getDynamicDispatchExtrinsicParams(
-      polkadotApi,
-      "recurring",
-    );
-    const { schedule, call } = extrinsicParams;
-    schedule.recurring.nextExecutionTime = null;
-
-    await expect(
-      automationTimeApi.buildScheduleDynamicDispatchTask(
-        keyringPair,
-        schedule,
-        call,
-      ),
-    ).rejects.toThrow();
-  },
-  DEFAULT_TIMEOUT_PER_TEST,
-);
-
-/**
- * dynamicDispatchTask.recurring with nextExecutionTime that is undefined fails
- */
-test(
-  "dynamicDispatchTask.recurring with nextExecutionTime that is undefined fails",
-  async () => {
-    const extrinsicParams = await getDynamicDispatchExtrinsicParams(
-      polkadotApi,
-      "recurring",
-    );
-    const { schedule, call } = extrinsicParams;
-    schedule.recurring.nextExecutionTime = undefined;
-
-    await expect(
-      automationTimeApi.buildScheduleDynamicDispatchTask(
-        keyringPair,
-        schedule,
-        call,
-      ),
-    ).rejects.toThrow();
-  },
-  DEFAULT_TIMEOUT_PER_TEST,
-);
-
-/**
- * dynamicDispatchTask.fixed schedule with execution time on non-o’clock(+5min) fails
- */
-test(
-  "dynamicDispatchTask.fixed schedule with execution time on non-o’clock(+5min) fails",
-  async () => {
-    const extrinsicParams = await getDynamicDispatchExtrinsicParams(
-      polkadotApi,
-      "fixed",
-    );
-    const { schedule, call } = extrinsicParams;
-    schedule.fixed.executionTimes[0] += 60 * 5;
-
-    // scheduler.buildScheduleDynamicDispatchTask will fail with invalid frequency
-    const extrinsicHex =
-      await automationTimeApi.buildScheduleDynamicDispatchTask(
-        keyringPair,
-        schedule,
-        call,
+    },
+    DEFAULT_TIMEOUT_PER_TEST,
+  );
+
+  /**
+   * dynamicDispatchTask.recurring with nextExecutionTime that is is a past time fails
+   */
+  it(
+    "dynamicDispatchTask.recurring with nextExecutionTime that is a past time fails",
```

Comment:
Added a test 
"dynamicDispatchTask.recurring with nextExecutionTime that is a past time fails"

Reply:
Are you sure the error is `InvalidTime`? I was able to send the transaction to on-chain and got a different error. `automationTime.PastTime`.

My schedule object,
```
Recurring: 
frequency: "600"
nextExecutionTime: "1,701,061,200"
```

Error from Network tab,
![CleanShot 2023-11-26 at 21 37 44](https://github.com/OAK-Foundation/oak.js/assets/2616844/dab49815-f50b-47b4-8176-87cb85934b94)

Reply:
Sorry.
If the past time is on a time slot, it will throw a `PastTime` error; otherwise, it will throw an `InvalidTime` error.